src/js/components/Tabs/Tabs.js

```diff
@@ -114,7 +83,21 @@ const Tabs = forwardRef(
           gap={theme.tabs.gap}
           {...tabsHeaderStyles}
         >
-          {tabs}
+          {React.Children.map(children, (child, index) => (
+            <TabsContext.Provider
+              // eslint-disable-next-line react/no-array-index-key
```

Comment:
This could be removed as well if the key={index} is removed. Missed this before.

```suggestion
```